message/data.go

```diff
@@ -1,76 +1,183 @@
+// message defines the received Json messages and their nested fields
 package message
 
+import (
+	"encoding/json"
+)
+
 type Data map[string]interface{}
 
+// []byte are automatically decoded from base64 when unmarshalled, while string (and json.RawMessage) are NOT
+
+/* potential enums, but doesn't typecheck in go, the checks must still be manual, so kinda useless
+type Object string
+const(
+	Lao Object = "lao"
+	Message Object = "message"
+	Meeting Object = "meeting"
+type Action string
+const(
+	Create Action = "create"
+	Update_properties Action = "update_properties"
+	State Action = "state"
+	WitnessKey Action = "witness"
+)*/
+
+
```

Comment:
can remove as it is implemented in parser. okay with you ?

Reply:
looks fine to me

Reply:
@Huntsekkern ?